scripts/st2bootstrap-deb.sh

```diff
@@ -100,13 +106,50 @@ install_st2web() {
   sudo service nginx restart
 }
 
+install_st2chatops {
+  # Install Node
+  curl -sL https://deb.nodesource.com/setup_4.x | sudo -E bash -
+  sudo apt-get install -y nodejs
+
+  # Install st2chatops
+  sudo apt-get install -y st2chatops
+}
+
+configure_st2chatops {
+  # Set credentials
+  sudo sed -i.bak -r "s/^(export ST2_AUTH_USERNAME.).*/\1$USERNAME/" /opt/stackstorm/chatops/st2chatops.env
+  sudo sed -i.bak -r "s/^(export ST2_AUTH_PASSWORD.).*/\1$PASSWORD/" /opt/stackstorm/chatops/st2chatops.env
+
+  # Setup adapter
+  if [ "$HUBOT_ADAPTER" == "slack" ] && [ -z "$HUBOT_SLACK_TOKEN" ]
```

Comment:
I wonder if we should even start this guessing game or just go straight to the warning. Without installer, we would have a very hard time trying to create proper user experience here.